packages/components/src/Modal/GenericModal/subcomponents/ModalFooter/ModalFooter.module.scss

```
@@ -0,0 +1,75 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/layout";
+
+.actions {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-start;
+}
+
+.actionsAlignStart {
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.padded {
+  padding: $spacing-md;
+  flex-direction: column;
+
+  @media (min-width: $layout-breakpoints-medium) {
+    padding: 0 $spacing-lg $spacing-lg $spacing-lg;
+    flex-direction: row-reverse;
+  }
+}
+
+.actionButton {
+  width: 100%;
+
+  @media (min-width: $layout-breakpoints-medium) {
+    width: inherit;
+  }
+}
+
+.actionButton + .actionButton {
+  margin-right: $spacing-sm;
+
+  @media (max-width: ( $layout-breakpoints-medium - 1px)) {
```

I think not doing this throws those funny errors right?
Although we could also leave these to fix later.

```suggestion
  @media (max-width: (calc(#{$layout-breakpoints-medium} - 1px))) {
```